CalPatRec/src/CalHelixFinderAlg.cc

```diff
@@ -1278,9 +1278,9 @@ namespace mu2e {
 // gianipez: procedure for aligning the phi vector
 //-----------------------------------------------------------------------------
     ::LsqSums4        szphi;
```

Comment:
Why is this class in the global namespace?

Reply:
I don't recall the reason TBH.

Reply:
Please make a separate PR that makes only the change of moving functions from the global to the mu2e namespace and updating the code where needed.